core-war/src/main/java/org/silverpeas/web/jobdomain/control/JobDomainPeasSessionController.java

```diff
@@ -2137,4 +2135,27 @@ private UserDetail getUser(String userId, UserDetail[] users) {
     }
     return null;
   }
-}
+
+  private void processDataOfExtraTemplate(String userId, HttpRequest request) {
+    PublicationTemplateManager templateManager = PublicationTemplateManager.getInstance();
+    PublicationTemplate template = templateManager.getDirectoryTemplate();
+    if (template != null) {
+      try {
+        PagesContext context = getTemplateContext(userId);
+        templateManager.saveData(template.getFileName(), context, request.getFileItems());
+      } catch (Exception e) {
+        MessageNotifier.addError("Les données du formulaire n'ont pas été enregistrées !")
+            .setDisplayLiveTime(10000);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Assign this magic number 10000 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS109)